File: Project/Domain/Url.cs
namespace Project.Domain
{
    public class Url
    {
        public int Id { get; set; }
        public string LongUrl { get; set; }
        public string ShortUrl {  get; set; }
    
        public Url(int id, string longUrl, string shortUrl)
        {
            Id = id;
            LongUrl = longUrl;
            ShortUrl = shortUrl;
        }
    }
}

File: src/main/java/com/mu/MicroUrl/Domain/Url.java
package com.mu.MicroUrl.Domain;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.UUID;

@Entity
@Table(name = "url")
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class Url {
    @Id
    @GeneratedValue
    private UUID Id;
    private String ShortUrl;
    private String OriginalUrl;
    private int Clicks;
}
